hypsrc-test/ref/src/Classes.html

```diff
@@ -194,7 +194,7 @@
 	><span class="annottext"
 	  >bar :: Int -&gt; Int
 </span
-	  ><a href="#"
+	  ><a href="Classes.html#bar"
```

Comment:
I like it, but I don't know why your patch could cause it.

Reply:
I have no idea either. It appeared recently after a rebase